lib/taurus/qt/qtgui/input/qwheel.py

```diff
@@ -845,6 +858,63 @@ def setShowArrowButtons(self, yesno):
     def resetShowArrowButtons(self):
         self.setShowArrowButtons(True)
 
+    def getHideEditWidget(self):
+        """getHideEditWidget(self) -> bool
+
+        Gets the info if edition widget should be hidden when 'focusOut' event
+        occurs.
+
+        @return (bool)
+        """
+        return self._hideEditWidget
+
+    def setHideEditWidget(self, focus_out=True):
+        """setFocusOut(self, focus_out=True) -> None
+
+        Sets if edition widget should be hidden when 'focusOut' event occurs.
+        If set to False, edition widget is hidden only when 'F2', 'Esc',
+        'Enter' and arrow button are pressed. Default set to True.
+
+        @param[in] focus_out (bool) whether or not to hide edition widget
+        after 'focusOut' event.
+        """
+        if focus_out and not self._hideEditWidget:
+            ed = self.getEditWidget()
+            ed.focusOut.connect(self.hideEditWidget)
+            self._hideEditWidget = True
+        elif not focus_out and self._hideEditWidget:
+            ed = self.getEditWidget()
+            ed.focusOut.disconnect(self.hideEditWidget)
+            self._hideEditWidget = False
+
+    def getForwardReturn(self):
```

Comment:
I propose a rename to be more consistent with Qt conventions:
`getForwardReturn` --> `isReturnForwarded`
`setForwardReturn` --> `setReturnForwarded`